src/sonic-supervisord-utilities-rs/src/bin/supervisor_proc_exit_listener.rs

```diff
@@ -0,0 +1,11 @@
+//! Supervisor process exit listener binary
+
+use sonic_supervisord_utilities_rs::supervisor_proc_exit_listener::main as supervisor_main;
+use std::process;
+
+fn main() {
```

Comment:
Why is there a `main()` method here and in `supervisor_proc_exit_listener` in the parent directory?

Reply:
Removed unused main() function in supervisor_proc_exit_listener.rs in the parent directory.